modules/videoio/src/cap_msmf.cpp

```diff
@@ -586,21 +702,29 @@ class CvCapture_MSMF : public cv::IVideoCapture
     virtual void close();
     virtual double getProperty(int) const CV_OVERRIDE;
     virtual bool setProperty(int, double) CV_OVERRIDE;
+    bool grabAudioFrame();
+    bool grabVideoFrame();
     virtual bool grabFrame() CV_OVERRIDE;
+    bool retrieveAudioFrame(int, cv::OutputArray);
+    bool retrieveVideoFrame(cv::OutputArray);
```

Comment:
> `cv::`

Should not be used in OpenCV code.

Exception is `cv::format()` (to avoid problem with `std::format()`)

Reply:
In master, cv:: is used with this argument. Is it necessary to remove it?

Reply:
> > `cv::`
> 
> Should not be used in OpenCV code.
> 
> Exception is `cv::format()` (to avoid problem with `std::format()`)

In master, cv:: is used with this argument. Is it necessary to remove it?

Reply:
remove from modified/added code.

No need to change unmodified parts.